src/containers/edit-profile/professional-info-tab/add-professional-category-modal/AddProfessionalCategoryModal.tsx

```diff
@@ -125,11 +125,18 @@ const AddProfessionalCategoryModal: FC<AddProfessionalCategoryModalProps> = ({
     initialValues: initialFormValues,
     onSubmit: formSubmission
   })
+
+  const isSubmitDisabled = (subjects: SubjectNameInterface[]) => {
+    const areSubjectsPresent = subjects.length > 0
+    const areAllSubjectsValid = subjects.every((subject) => subject._id)
+    return areAllSubjectsValid && areSubjectsPresent
+  }
+
```

Comment:
As for me, it`s better to move this to utils

Reply:
Thank you, good idea